.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports =  {
+    semi:  false,
+    trailingComma:  'all',
```

Comment:
Why? is not better without trailing commas?

Reply:
git diff is cleaner with

Reply:
I was mistaken, I was thinking in trailing semicolons. It's fine :)